packages/common/src/ui/layout/tables/utils/ColumnDefinitionSetBuilder.ts

```diff
@@ -82,7 +82,6 @@ const getColumnLookup = <T extends RecordWithId>(): Record<
   itemName: {
     key: 'itemName',
     label: 'label.name',
-    width: 300,
```

Comment:
this may cause layout issues in other pages - but if so, I couldn't find them. All looked ok to me

Reply:
I thought that adding a `maxWidth` would constrain the table on really wide screens.. but what it does is expand the width of the other columns such as location and unit. That's going to look a bit strange when the long item names are truncated and there's heaps of space around other columns.

I've added in a max width - but am still not sure we have the column sizing good in all scenarios.

Reply:
>  but am still not sure we have the column sizing good in all scenarios.

Yeah, I agree. I think putting a small-ish max-width on most columns would help, especially the ones with numbers, and then letting the free text fields grow, but only up to a point. But yeah, will take a bit of trial and error, and is quite time consuming to check all scenarios.

I'll make a note to notice when a column is looking weird and I'll take a screenshot. Then once we've got a decent number of these we can make some overall adjustments.